src/Sarif.Viewer.VisualStudio.ResultSources.Domain.Core/Entities/Secret.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sarif.Viewer.ResultSources.Domain.Entities
+{
+    /// <summary>
+    /// Represents a secret string.
+    /// </summary>
+    public class Secret
```

Comment:
>Secret [](http://example.com/codeflow?start=17&length=6)

what is the difference between entities.Secret and models.Secret?

Reply:
No difference in this case. It's a pattern to separate what comes from a source (entity) and what is used inside the component (model).